lightning-invoice/.travis.yml

```diff
@@ -1,31 +0,0 @@
-language: rust
-sudo: required
-rust:
-  - nightly
-  - beta
-  - stable
-cache: cargo
-
-jobs:
-  include:
-  - rust: 1.29.0
```

Comment:
You remove normal tests here but only seem to add fuzz tests later if I'm not mistaken.

Reply:
As a part of the workspace, its run automatically when you run `cargo test` in the root directory.